python/pipeline/pupil.py

```diff
@@ -870,38 +824,42 @@ def make(self, key):
 
             contours = (Tracking.ManualTracking & key).fetch(
                 'contour', order_by='frame_id ASC')
-            
-            # for manual tracking, we did not track eyelids, hence put -1.0 to be 
+
+            # for manual tracking, we did not track eyelids, hence put -1.0 to be
             # consistent with how we defined under PupilFitting.detect_visible_pupil_area
-            visible_portion = -1.0 
+            visible_portion = -1.0
 
             for frame_num in tqdm(range(nframes)):
 
                 if contours[frame_num] is None or len(contours[frame_num].squeeze()) < 3:
-                                        
-                    data_circle.append([*common_entry, frame_num, None, None, -3.0])
-                
+
```

Comment:
did the formatting add all of this space, what editor are you using, it looks a bit too much